build/version.go

```diff
@@ -41,7 +41,7 @@ func BuildTypeString() string {
 }
 
 // Intent: Major.Network.EvenReleases
```

Comment:
This intent comment is a bit off: 
'EvenReleases' should say something like 'Updates'

Reply:
Changed it to "Patch" as that was what we decided a while back.